Dockerfile.nginx

```diff
@@ -29,7 +31,7 @@ ENV \
   node="nodejs"
 RUN \
   apt-get -y update && \
-  apt-get -y install apt-transport-https ca-certificates && \
+  apt-get -y install apt-transport-https ca-certificates curl wget && \
```

Comment:
do we really need to install curl and wget? Doesn't seem related to the switch from python 3.5 to 3.6?

Reply:
probablty needed because they are no longer included by default?

Reply:
Indeed.